public/static/app/perf/perf.js

```diff
@@ -795,6 +795,7 @@ $http.get(templateUrl).success(function(template) {
     if (cedar && cedar.length) {
       scope.trendResults = scope.trendResults.concat(cedar);
     }
+    scope.trendResults = _.sortBy(scope.trendResults, (result) => Date.parse(result.create_time));
```

Comment:
Is create_time the revision create time or the result create_time?

What happens if a revision is rerun?

Reply:
I reran a revision and checked that sorting by order and create_time was the same (for a standard waterfall task).